src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -172,12 +160,113 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+                                    }
+                                    else if (profileSource == AzureGeneratorNamespace)
+                                    {
+                                        msgTarget.Text(RS_(L"SettingContainer_OverrideTargetAzureCloudShellGenerator"));
+                                    }
+                                    else
+                                    {
+                                        // TODO #1690: add special handling for proto-extensions here
+                                        // unknown generator; use fallback message instead
+                                        msgTarget.Text(RS_(L"SettingContainer_OverrideTarget"));
+                                    }
+                                    tb.Inlines().Append(msgTarget);
+                                }
+                                else
+                                {
+                                    // base layer
+                                    // TODO GH#3818: When we add profile inheritance as a setting,
+                                    //               we'll need an extra conditional check to see if this
+                                    //               is the base layer or some other profile
+                                    msgTarget.Text(RS_(L"Nav_ProfileDefaults/Content"));
+
+                                    Documents::Hyperlink hyperlink{};
+                                    hyperlink.Click([=](auto&, auto&) {
+                                        _NavigateHandlers(*this, settingSrc);
+                                    });
+                                    hyperlink.Inlines().Append(msgTarget);
+                                    tb.Inlines().Append(hyperlink);
+                                }
+                            }
+                            else
+                            {
+                                // unknown source; use fallback message instead
+                                msgTarget.Text(RS_(L"SettingContainer_OverrideTarget"));
+                                tb.Inlines().Append(msgTarget);
+                            }
+                        }
+
+                        // Append Message Suffix
+                        {
+                            Documents::Run msgSuffix{};
+                            msgSuffix.Text(templateText.substr(pos + 2, templateText.npos));
+                            tb.Inlines().Append(msgSuffix);
+                        }
+                    }
+                    else
+                    {
+                        // '{}' was not found
+                        // fallback to more ambiguous version
+                        hstring overrideMsg{ fmt::format(std::wstring_view{ RS_(L"SettingContainer_OverrideIntro") }, RS_(L"SettingContainer_OverrideTarget")) };
+                        tb.Text(overrideMsg);

```

Comment:
This should only happen if the localization was messed up. Idk if there's a way to get around this? Or handle it better?